File: money/fallspring.py
fallBalance = 6299.37
nonResidency = 4709.00
residency = fallBalance - nonResidency
fallLoan = 18250.00
nonResidencyRefund = fallLoan - nonResidency
residencyRefund = fallLoan - residency

primeFamily = 16.00
primeStudent = 8.00
nitro = 4.99
spotify = 5.99
dashPass = 4.99
gamepass = 16.19
monthlySubscriptions = primeFamily + primeStudent + nitro + spotify + dashPass + gamepass

rent = 840
internet = 50
pet = 25
laundry = 35
utilities = 50
rentersInsurance = 15.34
monthlyApartment = rent + internet + pet + laundry + utilities + rentersInsurance

monthly = monthlyApartment + monthlySubscriptions

fallSemester = monthly * 5

nonResidencyRemainder = nonResidencyRefund - fallSemester
residencyRemainder = residencyRefund - fallSemester

print("Your student balance for the Fall is " + str(round(fallBalance, 2)) + "\n")
print("The cost of tuition for being a non-resident of Ohio is " + str(round(nonResidency, 2)) + "\n")
print("Your student balance for the Fall with residency is " + str(round(residency, 2)) + "\n")
print("Your refund for being a non-resident will be " + str(round(nonResidencyRefund, 2)) + "\n")
print("Your refund with residency will be " + str(round(residencyRefund, 2)) + "\n")
print("Your monthly apartment expenses costs you " + str(round(monthlyApartment, 2)) + "\n")
print("Overall, your monthly expenses will be " + str(round(monthly, 2)) + "\n")
print("Your total expneses for the fall semester will be " + str(round(fallSemester, 2)) + "\n")
print("Your remaining money for the fall semester as a non-resident will be " + str(round(nonResidencyRemainder, 2)) + "\n")
print("Your remaining money for the fall semester as an Ohio resident will be " + str(round(residencyRemainder, 2)) + "\n")

File: money/fallspring.java
public class fallspring {

    public static void main(String[] args) {
        double fallBalance = 9924.37;
        double nonResidency = 4709.00;
        double residency = fallBalance - nonResidency;
        double fallAid = 18493.00;
        double nonResidencyRefund = fallAid - fallBalance;
        double residencyRefund = fallAid - residency;

        double primeFamily = 16.00;
        double primeStudent = 8.00;
        double nitro = 4.99;
        double spotify = 5.99;
        double dashPass = 4.99;
        double gamepass = 16.19;
        double monthlySubscriptions = primeFamily + primeStudent + nitro + spotify + dashPass + gamepass;

        double rent = 840.00; //1060.00
        double internet = 50.00;
        double pet = 25.00;
        double laundry = 35.00;
        double utilities = 50.00;
        double rentersInsurance = 15.34;
        double monthlyApartment = rent + internet + pet + laundry + utilities + rentersInsurance;

        double monthly = monthlyApartment + monthlySubscriptions;

        double fallSemester = monthly * 5;

        double nonResidencyRemainder = nonResidencyRefund - fallSemester;
        double residencyRemainder = residencyRefund - fallSemester;

        System.out.println("Your student balance for the Fall is " + round(fallBalance));
        System.out.println("The cost of tuition for being a non-resident of Ohio is " + round(nonResidency));
        System.out.println("Your refund for being a non-resident will be " + round(nonResidencyRefund));
        System.out.println("Your refund with residency will be " + round(residencyRefund));
        System.out.println("Your monthly subscriptions costs you " + round(monthlySubscriptions));
        System.out.println("Your monthly apartment expenses costs you " + round(monthlyApartment));
        System.out.println("Overall, your monthly expenses will be " + round(monthly));
        System.out.println("Your total expenses for the fall semester will be " + round(fallSemester));
        System.out.println("Your remaining money for the fall semester as a non-resident will be " + round(nonResidencyRemainder));
        System.out.println("Your remaining money for the fall semester as an Ohio resident will be " + round(residencyRemainder)); // - (840 * 2)
        System.out.println("Reaminimg money if you switch apartments: " + round(residencyRemainder - (840 * 2) - 250));
    }   

    public static String round(double value) {
        return String.format("%.2f", value);
    }
}